### ISSUE TITLE
Update theme selector to include system host

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#20100]
**Is your feature request related to a problem? Please describe.**
Now that we have a system theme that lives in the system host, we need to be able to select it from both theme selectors (dropdown and dialog)

**Describe the solution you'd like**
We can pass a `system` input to the site selector component that will allow selecting the system host from that selector. That will do for most of the part.

The other part will be:

1. We show the themes from the current host
2. If the current host has no themes
3. We auto-select the system host so the user can select at least the system theme.

**Context**
We have 2 theme selectors:

```
apps/dotcms-ui/src/app/view/components/dot-edit-layout-designer/components/dot-theme-selector/dot-theme-selector.component.ts
```

and

```
apps/dotcms-ui/src/app/view/components/dot-theme-selector-dropdown/dot-theme-selector-dropdown.component.ts
```

We need to do this in both.

**Comment by hmoreras** [comment]
PR: https://github.com/dotCMS/core-web/pull/1632

**Comment by ivorpad** [comment]
Tested on core

<img width="300" alt="Screenshot 2021-04-05 at 23 03 51" src="https://user-images.githubusercontent.com/52452/113627143-5eb85700-9663-11eb-8c14-0ad8251437c9.png">

<img width="300" alt="Screenshot 2021-04-05 at 23 01 32" src="https://user-images.githubusercontent.com/52452/113627152-61b34780-9663-11eb-8132-71db18efd62d.png">

**Comment by bryanboza** [comment]
Fixed, tested on release-21.05 // Postgres // FF
